analyzer/src/main/kotlin/PackageCurationProvider.kt

```diff
@@ -21,6 +21,29 @@ package org.ossreviewtoolkit.analyzer
 
 import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.utils.common.ConfigurablePluginFactory
```

Comment:
> as a preparation for making `PackageCurationProvider`s plugins.

Strictly speaking this is a bit misleading: `PackageCurationProvider`s themselves will never become plugins, but only their factories, and the factories become plugins already in this commit, so this is not a "preparation".

Reply:
I have removed that part of the sentence.